web/src/Web.App/Controllers/TrustSpendingController.cs

```diff
@@ -40,10 +40,7 @@ public async Task<IActionResult> Index(
 
                 var trust = await establishmentApi.GetTrust(companyNumber).GetResultOrThrow<Trust>();
                 var schoolsQuery = new ApiQuery();
-                foreach (var school in trust.Schools)
-                {
-                    schoolsQuery.AddIfNotNull("urns", school.URN);
-                }
+                schoolsQuery.AddIfNotNull("companyNumber", companyNumber);
```

Comment:
does phase need to be added here for the later call to metricRagRatingApi?

Reply:
I've removed the dependency on `OverallPhase` when querying for a Trust as it was causing unnecessary complexity and unlike the XHR requests on the React charts, we never perform additional filters by Phase on the Trust pages. Also adding in the missing E2E tests for this page, as this omission should have been picked up by a failing test.